modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/UpdateObjectEntryObjectActionExecutorImpl.java

```diff
@@ -65,6 +65,9 @@ public void execute(
 
 		TransactionCommitCallbackUtil.registerCallback(
 			() -> {
+				ObjectEntryThreadLocal.setSkipReadOnlyObjectFieldsValidation(
```

Comment:
Is it necessary to set to true value here? I saw that you also do this inside `_execute` method.